### ISSUE TITLE
Minor question: why use b_batch?

### ISSUE THREAD

**Issue by abarajithan11** [experiencor/keras-yolo2#407]
The input and output to the network are ([image_batch, b_batch], y_batch). However, the b_batch is neither in loss function, the model, prediction nor anywhere else. For prediction we give a dummy array as b_batch. Then, what is purpose of batching it, sending it into the model for training..etc?

Why can't we simply have (image_batch, y_batch) as the input, output of the network?

**Comment by hokim72** [comment]
b_batch is fed to  true_boxes
true_boxes is used for masks for confidence and class in loss function

**Comment by mumuyanyan** [comment]
> b_batch is fed to true_boxes
> true_boxes is used for masks for confidence and class in loss function

In the training,there is already true_boxes.Sorry,i just don't get it.

**Comment by speeding-motor** [comment]
have you slove it now, I can not get it too
